### ISSUE TITLE
Currency converter is not working

### ISSUE THREAD

**Issue by trinaldi** [awamper/google-calculator#9]
Apparently, Google changed their converter URL. [This StackOverflow thread](https://stackoverflow.com/questions/49324112/google-finance-converter-stopped-working-or-changed-its-url) from March/2018 shows a possible workaround.

**Comment by awamper** [comment]
Hello, i checked provided link, the workaround doesn't seem to work.

**Comment by trinaldi** [comment]
Indeed, I've also tried it.

Seems like we're stuck

**Comment by prashu20** [comment]
Yes, i use the gnome extension on Ubuntu 18.04. Looks good but Currency Converter is not working. Any way to fix it?

**Comment by trinaldi** [comment]
@prashu20 That's a problem with the Google API, not the plugin directly.
